client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -10,25 +10,46 @@ import { localize } from 'i18n-calypso';
  */
 import PurchaseDetail from 'components/purchase-detail';
 import { localizeUrl } from 'lib/i18n-utils';
+import { isEnabled } from 'config';
 
 /**
  * Image dependencies
  */
 import paymentsImage from 'assets/images/illustrations/payments.svg';
 
 export default localize( ( { isJetpack, translate } ) => {
+	let supportDocLink;
+	if ( isEnabled( 'earn/rename-payment-blocks' ) ) {
+		supportDocLink = localizeUrl(
+			isJetpack
+				? 'https://jetpack.com/support/pay-with-paypal/'
+				: 'https://wordpress.com/support/pay-with-paypal/'
+		);
+	} else {
+		supportDocLink = localizeUrl(
+			isJetpack
+				? 'https://jetpack.com/support/simple-payment-button/'
```

Comment:
Hm, we have a Simple Payment (singular) Button, and a Simple Payments (plural) Block – should we align these? cc/ @lauac

Reply:
They'll be unused once the rename is live and therefore can be removed then, so I wouldn't worry to much about that.